app/models/trainee.rb

```diff
@@ -337,6 +337,8 @@ class Trainee < ApplicationRecord
 
   scope :on_early_years_routes, -> { where(training_route: EARLY_YEARS_TRAINING_ROUTES.keys) }
 
+  scope :without_placements, -> { where.not(id: joins(:placements).select(:id)) }
```

Comment:
Does `where.not(id: Placement.select(:trainee_id))` also work (avoiding the join)?

Reply:
It does indeed, have replaced it with yours - thanks